collect_app/src/androidTest/java/org/odk/collect/android/support/StubOpenRosaServer.java

```diff
@@ -140,6 +141,10 @@ public void removeHashInFormList() {
         noHashInFormList = true;
     }
 
+    public void omitOpenRosaHeader() {
```

Comment:
This isn't being used anywhere as far as I can see?

Reply:
Whoops, thank you! I incompletely backed out of having an Espresso test for the legacy form list case. Will fix.